### PR TITLE
Updates database to postgres

### PR DESCRIPTION
**Pull request by AdamM-FC**
Updates the database from sqlite to postgres

### CODE REVIEW

File: rickmorty-api/.gitignore [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
```
Comment (mugx-fc) [inline rickmorty-api/.gitignore:1]: why are you adding this file ?

**Review by mugx-fc — CHANGES_REQUESTED** [review]

**Review by mugx-fc — APPROVED** [review]
lgtm 👍

File: config/database.yml [hunk 2]
```diff
@@ -1,25 +1,25 @@
-# SQLite. Versions 3.8.0 and up are supported.
-#   gem install sqlite3
+# Postgres. Versions 3.8.0 and up are supported.
```
Comment (MassimoPolimeni) [inline config/database.yml:1]: can we update this comment? 🙏 just to align everything

**Review by MassimoPolimeni — APPROVED** [review]
LGTM, just left a nitpick

File: config/database.yml (hunk 2 above)
Comment (AdamM-FC) [reply config/database.yml:1]: Updated

**Pull request merged**